### ISSUE TITLE
Use "coordinate" (singular form) on referencing single coordinate

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#11390]
### Tell us your idea!

In c:geo we have mostly the name "coordinates" on places where only a single coordinate is referenced.
This should be changed to "coordinate".

I propose to only change the text and let the keyword as is. 

related to #11354 Point 2.

**Comment by bekuno** [comment]
As I removed the string changes from PR #11372 let add also the Keyword single form to this Issue.
I change the title according.

from Issue #11354 :

**2)** Coordinates vs. Coordinate
The word "Coordinates" is in c:geo used on many places, not only in the search GUI.
The word Coordinates is the plural form of Coordinate. And mostly we mean a single coordinate (consisting of latitude and longitude). And we search in the search GUI by a single coordinate.
So lets use the single form here and check also other places.

**3)** Keywords vs. Keyword
The headline says "Keywords", the helper text and the button "Keyword".
The headline should use the single form (as the other headlines).
Can we really search for multiple keywords? (I did not tested it). If yes then use "Keyword(s)".

**Comment by rsudev** [comment]
Regarding coordinate/coordinates I have a different feeling. According to my understanding you need two coordinates (lat and long) to define a point in the two-dimensional topology the earth is described with, analogous to the x/y-coordinates to define any position in a cartesian coordinate system. So I would keep the plural form (as e.g. the search box on geocaching.com does and many other places on the interweb...)

**Comment by bekuno** [comment]
I changed the issue description to only cover the coordinates/coordinate problem.
The keywords is changed in #11745 .

**Comment by ztNFny** [comment]
Plural is the proper use. "coordinate" would be "N0", "coordinates" is "N0 E0". 2 coordinates define a point on a plain, 3 coordinates define a point in space. With geocaching we're usually (always) talking about the point on plain situation. 

From [Wikipedia](https://en.wikipedia.org/wiki/Coordinate_system): "In geometry, a coordinate system is a system that uses one or more numbers, or **coordinates**, to uniquely determine the position of the points or other geometric elements on a manifold such as Euclidean space."

From [Wiktionary](https://en.wiktionary.org/wiki/coordinate#English): "Give me your **coordinates** and we'll come and rescue you."

From [Oxford Dictionary](https://www.oxfordlearnersdictionaries.com/definition/english/coordinate_2): "**either of two numbers** or letters used to fix the position of a point on a map or graph"

**Comment by moving-bits** [comment]
Following the discussion I vote for closing this issue.

**Comment by bekuno** [comment]
I searched for a definition especially for German. But I did not found good results. Not in the Duden, not for mathematics.
Sometime there are results in the singular form, but many results refers to many points.
At least a definition for using "Koordinaten" for a point in 2 or 3 dimension is found here: https://www.dwds.de/wb/wdg/Koordinate
Therefore it is fine to close here and use coordinates/Koordinaten everywhere in c:geo for points in the plane.
